### ISSUE TITLE
find a bug in v4.0.14 and v4.0.15: --save-path command line option is broken

### ISSUE THREAD

**Issue by ybyygu** [martyr-deepin/deepin-screenshot#30]
The cmdline option to set output file path is broken:

> deepin-screenshot -s /tmp/a.png
> ls /tmp/a.png
ls: cannot access '/tmp/a.png': No such file or directory

It works if not specifiying the saving path.

The last good version is 4.0.11.

System: Arch Linux, 4.16.13-1-zen

**Comment by ybyygu** [comment]
looks like it already got fixed:

https://github.com/linuxdeepin/deepin-screenshot/commit/85b51d63cc845dd67e8bf8c77a8cdf1b6d2b75ce

waiting for the new release

**Comment by ybyygu** [comment]
not fixed actually...,  the bug is still there :(

What I tried:
1. update to v4.0.15 using Archlinux package: deepin-screenshot-4.0.15-1-x86_64.pkg.tar.xz
2. built the binary from git source codes.

None of them works as expected.

```
> ./deepin-screenshot -s /tmp/a.png 
> ll /tmp/a.png
ls: cannot access '/tmp/a.png': No such file or directory
```

git status:
[ybyygu@ARCHTower] ~/Incoming/scratch/deepin-screenshot 16:49:53
> git status
On branch master
Your branch is up to date with 'origin/master'.

**Comment by ybyygu** [comment]
fixed in v4.0.16
